src/realtime/price.updates.ts

```diff
@@ -209,35 +198,40 @@ export class PriceUpdates {
         const that = this;
 
         setTimeout(() => {
-            if (!isEmpty(that.subscribersWithTicker)) {
-                that.subscribersWithTicker.forEach((symbolTicker) => {
-                    log(`cancelMktData ${symbolTicker.symbol} tickerId=${symbolTicker.tickerId}`);
-                    that.ib.cancelMktData(symbolTicker.tickerId);
-                });
+            for (const symbolTicker of Object.values(this.tickerIdToData)) {
```

Comment:
I think we don't need this right?

Reply:
I'm not sure which lines you're referring to exactly as being unnecessary?